azure/src/wizard/ResourceGroupListStep.ts

```diff
@@ -46,6 +45,9 @@ export class ResourceGroupListStep<T extends types.IResourceGroupWizardContext>
         // Cache resource group separately per subscription
         const options: IAzureQuickPickOptions = { placeHolder: 'Select a resource group for new resources.', id: `ResourceGroupListStep/${wizardContext.subscriptionId}` };
         wizardContext.resourceGroup = (await wizardContext.ui.showQuickPick(this.getQuickPicks(wizardContext), options)).data;
+        if (wizardContext.resourceGroup) {
+            await LocationListStep.setLocation(wizardContext, nonNullProp(wizardContext.resourceGroup, 'location'));
```

Comment:
Discussion question...

Should we always set location whenever we pick a resource group, or should it only be set if `_location` is undefined?  Whenever we go through the flow to create a resource group, we automatically select and set the location, so I'm kind of leaning towards `option A` but could see both ways being feasible but having different side effects.

A)
``` javascript
if (wizardContext.resourceGroup) {
  ...
}
``` 
OR

B)
```javascript
if (wizardContext.resourceGroup && !LocationListStep.hasLocation(wizardContext)) {
  ...
}
```

Reply:
I think I prefer option B a little bit better. If we set the location ahead of time for whatever reason, I'd prefer to respect those settings.